client/executor/src/native_executor.rs

```diff
@@ -314,143 +286,6 @@ impl<D: NativeExecutionDispatch> RuntimeInfo for NativeExecutor<D> {
 	}
 }
 
-/// Helper inner struct to implement `RuntimeSpawn` extension.
-pub struct RuntimeInstanceSpawn {
```

Comment:
Moved to 'sp_tasks' pool_spawn.rs (with additional fields).
Also some logic from it use in variant in 'sp_tasks' inline_spawn.rs